kitsune/users/auth.py

```diff
@@ -51,6 +51,30 @@ def get_token(self, payload):
         self.refresh_token = token_info.get("refresh_token")
         return token_info
 
+    @classmethod
+    def refresh_access_token(cls, refresh_token):
```

Comment:
probably want a `ttl` argument here, currently we just request a 5-minute access token (so it expires shortly after the profile update task completes), but in other scenarios we may want a longer-lived token